core/src/main/scala/iliad/gl/egl.scala

```diff
@@ -32,6 +32,17 @@ object Attributes {
   def empty[K <: IntConstant, V <: IntConstant]: Attributes[K, V] = apply()
 }
 
+sealed trait EGLError
```

Comment:
These may be better in a main errors.scala file, where they can be subclasses of `IliadError`